helpers/cdn.js

```diff
@@ -2,21 +2,37 @@
 
 const cdnify = require('./lib/cdnify');
 const {addResourceHint} = require('./lib/resourceHints');
+const utils = require("./3p/utils");
+
 
 const factory = globals => {
+    function addHint(fullPath, options) {
+        return addResourceHint(
+            globals,
+            fullPath,
+            options.hash.resourceHint,
+            options.hash.as
+        );
+    }
+
     const cdn = cdnify(globals);
 
     return function (path) {
         let fullPath = cdn(path);
 
         const options = arguments[arguments.length - 1];
         if (options.hash && options.hash.resourceHint) {
-            fullPath = addResourceHint(
-                globals,
-                fullPath,
-                options.hash.resourceHint,
-                options.hash.as
-            );
+            if (!utils.isString(fullPath)) {
+                console.warn(`Skipping resource hint creation in CDN fill path was not valid [${fullPath}]`);
```

Comment:
I think if these are logging out to console as WARN in renderer it will trigger a sentry alert, do we want this in this case?

Reply:
only in the `earlyHint` helper case. will downgrade all `warn` no `info` tho.